tests/providers/amazon/aws/auth_manager/views/test_auth.py

```diff
@@ -69,7 +69,7 @@ def aws_app():
         ) as mock_is_policy_store_schema_up_to_date:
             mock_is_policy_store_schema_up_to_date.return_value = True
             mock_parser.parse_remote.return_value = SAML_METADATA_PARSED
-            return application.create_app(testing=True)
+            return application.create_app(testing=True, config={"WTF_CSRF_ENABLED": False})
```

Comment:
Why was this change required here?

Reply:
Flask require a csrf token when submitting a post request. Setting this constant to `False` disable this check. Which we want for tests :)